Detectors/MUON/MCH/Raw/ElecMap/src/Mapper.cxx

```diff
@@ -10,6 +10,9 @@
 // or submit itself to any jurisdiction.
 
 #include "MCHRawElecMap/Mapper.h"
+#include "MCHConstants/DetectionElements.h"
+#include <optional>
+#include <unordered_map>
```

Comment:
I suppose these 3 includes are not needed anymore

Reply:
indeed